Apps/Playground/0.65/android/app/proguard-rules.pro

```diff
@@ -0,0 +1,10 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in /usr/local/Cellar/android-sdk/24.3.3/tools/proguard/proguard-android.txt
+# You can edit the include path and order by changing the proguardFiles
+# directive in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# Add any project specific keep options here:
```

Comment:
A couple lines are needed here to make Hermes work. Probably worth copying those over from the 0.64 project.

Reply:
Done with documentation update as well.